mod.ts

```diff
@@ -28,17 +28,16 @@ function hmacSha256Hex(key: string | ArrayBuffer, data: string): string {
 }
 
 function ymd(date: Date): string {
-  return date.toISOString().substring(0,10).replace(/[^\d]/g, '')
+  return date.toISOString().substring(0, 10).replace(/[^\d]/g, '')
 }
 
 function isoDate(date: Date): string {
-  return `${date.toISOString().substring(0,19).replace(/[^\dT]/g, '')}Z`
+  return `${date.toISOString().substring(0, 19).replace(/[^\dT]/g, '')}Z`
 }
 
 function parseOptions(provided: GetSignedUrlOptions): Required<GetSignedUrlOptions> {
-  if (!provided.objectPath.startsWith('/')) {
-    provided.objectPath = '/' + provided.objectPath
-  }
+  const fixedPath = `/${provided.path}`.replaceAll('//', '/')
```

Comment:
Nice. Though a little more difficult to understand intent at first glance.

Reply:
I would argue that control statements are harder to read. Without them the flow of the code is somewhat "uninterrupted" and easier to read.

Reply:
I can get onboard with that :)